### ISSUE TITLE
Build failing on Travis CI

### ISSUE THREAD

**Issue by Agent-Isai** [isainode/Plex#1]
The build is failing on Travis CI. I haven't tested it out but out good friend @AlphaT3ch might help us :)

**Comment by JeDa** [comment]
Hey Angry ork @AlphaT3ch, where the jerk have you been? Get back, sir! The party needs you. Argh, I can't live without you {{needs citation}}! Get here, @AlphaT3ch.

**Comment by Agent-Isai** [comment]
@JeDa k

**Comment by Agent-Isai** [comment]
@JeDa PM me on fn

**Comment by coca-koala** [comment]
yoyyOyoo!! @AlphaT3ch WerE IZZ UU Maynn

**Comment by JeDa** [comment]
@Agent-Isai no

**Comment by JeDa** [comment]
@AlphaT3ch come back ;_;

**Comment by Agent-Isai** [comment]
@JeDa @coca-koala He's ignoring us ;___; @AlphaT3ch will probably never come back although he's on freenode as AlphaTech but he's offline

**Comment by coca-koala** [comment]
@Agent-Isai @JeDa why da fuk is dis nigga @AlphaT3ch reactivatin his znc? it was told dat he can't irc till 18 yo. He should liek make a statement or somethin about wat da fuk's up.

**Comment by AlphaT3ch** [comment]
@Agent-Isai Not ignoring... Also, that wasn't a BC @coca-koala :p

**Comment by Agent-Isai** [comment]
Hey there @AlphaT3ch, also, fuck!

**Comment by Agent-Isai** [comment]
@AlphaT3ch Oh, Node.js, never mind I'm ok :)

**Comment by JeDa** [comment]
@AlphaT3ch HELLO MAN!!!1
HOW HAVE YOU BEEN

**Comment by coca-koala** [comment]
@AlphaT3ch shiit man. How Ya been? Ya Comin back to irc any time soon yo?

**Comment by coca-koala** [comment]
@AlphaT3ch Yo also ur nick got yanked nigga. Dat totes was a bnc too. Merry Christmas, New Years bra.

**Comment by Agent-Isai** [comment]
@AlphaT3ch @coca-koala @JeDa I took your nick cuz frackster got one of his oper friends to drop ur nick give it to him. He owned all your channels before but since I forced him to drop the nick, you haz no channels

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by frackster** [comment]
That's not true lol. I was told you're not coming back to IRC until you're 18 so a freenode oper did drop the nick as I'm collecting *Tech nicknames. I grouped it, but when you rejoined a couple days ago I dropped it and notified you so you could re-register it. You never re-registered the nick so Agent_Isai chose to group it to his account. As a result of the drop, some select channels were assigned founder to me. However, if you ever come back I will gladly reassign founder to you.

**Comment by Agent-Isai** [comment]
@AlphaT3ch If you need your IRC nick, you can tell me so I can drop it

**Comment by Agent-Isai** [comment]
I registered it for safe keeping against frackster

**Comment by AlphaT3ch** [comment]
test32

**Comment by frackster** [comment]
What Isai said is not the whole truth. :P However, you should've been emailed my response I said previously despite it having been deleted multiple times (as I'm sure this one eventually will be as well). I don't even know why you continue to trust Isai, everyone knows he's a chronic liar and manipulator and he's the last person you should trust. Do so at your own risk I suppose.

**Comment by frackster** [comment]
Also let me know if you want the channels that were founder'd to me back... TheCasino timed out a while ago btw.
